src/www/system_advanced_misc.php

```diff
@@ -397,6 +402,28 @@ function thermal_modules()
                 </td>
               </tr>
               <tr>
+                <td><i class="fa fa-info-circle text-muted"></i>  <?=gettext('On Battery Normal Mode') ?></td>
```

Comment:
It's my first time editing php...
I thought the comment should be added to add some explanation for the normal option.